src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -324,8 +330,9 @@ export function registerTerminalActions() {
 			});
 		}
 		async run(accessor: ServicesAccessor) {
-			accessor.get(ITerminalGroupService).activeGroup?.focusNextPane();
-			await accessor.get(ITerminalGroupService).showPanel(true);
+			const terminalGroupService = accessor.get(ITerminalGroupService);
```

Comment:
when I've focused a terminal editor w the panel closed and run this command, the panel opens. is that what we want?

or should we check if `terminalService.activeInstance.target === editor` and if so, return early?

Reply:
Yes I think this is fine and it's also the existing behavior. My thinking is if you only have a terminal editor and run the command to focus the next _panel_ terminal, it's fine to show the panel and create one.